core/src/main/java/com/graphhopper/routing/RoundTripAltAlgorithm.java

```diff
@@ -209,6 +212,17 @@ public void setWeightLimit( double weightLimit )
     }
 
     @Override
+    public void setMaxVisitedNodes( int numberOfNodes )
+    {
+        this.maxVisitedNodes = numberOfNodes;
+    }
+
+    protected boolean isMaxVisitedNodesExceeded()
+    {
+        return maxVisitedNodes > getVisitedNodes();
```

Comment:
but not important to add a test as we have your nice PR soon and throw this away :)